packages/rocketchat-lib/server/models/Messages.js

```diff
@@ -284,9 +284,11 @@ RocketChat.models.Messages = new class extends RocketChat.models._Base {
 		return this.findOne(query);
 	}
 
-	getLastMessageSentWithNoTypeByRoomId(rid) {
+	getLastVisibleMessageSentWithNoTypeByRoomId(rid, messageId) {
 		const query = {
+			_id: { $ne: messageId },
```

Comment:
I assume this is safe?  if this is undefined will this have negative effects on this query?